app/views/estateagentbusiness/business_servicess.scala.html

```diff
@@ -40,14 +40,14 @@
             @errorGroup(
                 field = f("services")
             ) {
-                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.06", value = "06")
+                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.01", value = "01")
+                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.02", value = "02")
                 @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.03", value = "03")
+                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.04", value = "04")
                 @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.05", value = "05")
-                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.02", value = "02")
-                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.08", value = "08")
+                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.06", value = "06")
                 @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.07", value = "07")
-                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.04", value = "04")
-                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.01", value = "01")
+                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.08", value = "08")
                 @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.09", value = "09")
             }
```

Comment:
here you've changed the order of checkboxes, are the services then in alphabetical order?

Reply:
Will fix in 4928